torch/csrc/jit/codegen/cuda/scheduler/reduction.cpp

```diff
@@ -649,44 +763,47 @@ ReductionParams OuterReductionHeuristic(
     }
   }
 
-  // Cannot unroll with cross grid reductions
-  if (grdim > 1 && iter_unroll_factor > 1) {
-    // Readjust the thread bindings, ideally we would repeat the block setup
-    // without considering iter domain unrolling, but for now will simplify
-    bdimx = std::min(max_threads_in_block, bdimx * iter_unroll_factor);
-    // Round bdimx down to power of 2 or multiple of warp
-    bdimx = bdimx > warp_size ? bdimx - bdimx % warp_size
-                              : scheduler_utils::lastPow2(bdimx);
-    // bdimy can only be reduced here from before
-    bdimy = std::max(max_threads_in_block / bdimx, (int64_t)1);
-    // Reset iteration unroll
-    iter_unroll_factor = 1;
-  }
-
   int64_t gdimx = LaunchParams::UNINITIALIZED_VAL;
   int64_t gdimy = LaunchParams::UNINITIALIZED_VAL;
 
+  // In these instances latency of the cleanup may be significant so flip gdimx
+  // and gdimy to try and prevent all cleanup from happening at the same time
+  bool flip_grid = gidim > 1 && gidim < 8;
+  flip_grid = false;
```

Comment:
So, currently, are we just always disabling this?

Reply:
Yes, it is piped through correctly to use, but I haven't used it yet as I didn't see any cases benefit. However, I didn't check if any cases benefit after tuning the heuristics, I want to give it a try when I have a chance then will cleanup if it's not useful on outer reductions.